test/test_hl_api.py

```diff
@@ -168,7 +166,7 @@ def test_speedup_parallel_evolve(population_params, genome_params, ea_params):
     # Serial execution
 
     for n_processes in [1, 2, 4]:
-        pop = cgp.Population(**population_params, genome_params=genome_params)
+        pop = cgp.Population(**population_params, genome_params=[genome_params])
```

Comment:
why does this need to be a list of params?

Reply:
It doesn't. I reverted the change.